src/parquet_builder/sigma.jl

```diff
@@ -16,13 +16,15 @@
 - A DataFrame with fields `:type`, `:extT`, `:diagram`, `:hash`
 - All sigma share the same incoming Tau index, but not the outgoing one
 """
-function sigma(para::DiagPara{W}, extK=DiagTree.getK(para.totalLoopNum, 1), subdiagram=false; name=:Σ, resetuid=false, blocks::ParquetBlocks=ParquetBlocks()) where {W}
+function sigma(para::DiagPara{W}, extK=DiagTree.getK(para.totalLoopNum, 1), subdiagram=false;
+    name=:Σ, resetuid=false, blocks::ParquetBlocks=ParquetBlocks()
+) where {W}
     resetuid && uidreset()
     (para.type == SigmaDiag) || error("$para is not for a sigma diagram")
     (para.innerLoopNum >= 1) || error("sigma must has more than one inner loop")
     # @assert length(extK) == para.totalLoopNum
     # @assert (para.innerLoopNum <= 1) || ((NoBubble in para.filter) == false) "The many-body correction sigma only accounts for half of the bubble counterterm right now."
-    if (para.innerLoopNum > 1) && (NoBubble in para.filter)
+    if (para.innerLoopNum > 1) && (NoBubble in para.filter) && (compact == false)
```

Comment:
Is `compact` defined? I cannot find it in the codebase.

Reply:
I forget to delete it. Thank you for pointing it out